docs/features.rst

```diff
@@ -169,7 +169,49 @@ Multiple Connections
   multi-connection setup.
 - SQL Explorer also supports user-provided connections in the form
   of standard database connection details, or uploading CSV, JSON or SQLite
-  files. See the 'User uploads' section of :doc:`settings`.
+  files.
+
+File Uploads
```

Comment:
@marksweb If you have a moment, I'd love for you to give this a read and let me know what you think. Not asking for a full code review, just looking over the docs and make sure the functionality is clear to you. If there are outstanding questions, I can address them in the docs, or perhaps just by reading the docs you will think of some edge case or test I haven't thought of! I would welcome a brain-dump from you on this functionality.